enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/MobileTokenController.java

```diff
@@ -62,6 +63,9 @@ public class MobileTokenController {
 
     private static final Logger logger = LoggerFactory.getLogger(MobileTokenController.class);
 
+    // Disallowed flags contain onboarding flags used before onboarding process is finished
+    private static final List<String> DISALLOWED_FLAGS = Arrays.asList("VERIFICATION_PENDING", "VERIFICATION_IN_PROGRESS");
```

Comment:
You may use `List#of`, as bonus, you get immutable collection.

Reply:
Right, we moved away from Java 8 support, I forgot.